.reuse/dep5

```diff
@@ -0,0 +1,122 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: KTX-Software
+Source: https://github.com/KhronosGroup/KTX-Software
+
+Files: tests/srcimages/GoldenGateBridge3/* tests/srcimages/Yokohama3/*
+Copyright: Emil Persson http://www.humus.name
+License: CC-BY-3.0
+
+Files: tests/srcimages/Iron_Bars/*
+Copyright: Katsuagi https://3dtextures.me
+License: CC0-1.0
+
+# Jon - Unclear whether all remaining images & icons fall under the Khronos license
+Files: */*.png */*.ktx */*.ico */*.icns */*.jpg */*.pam */*.pgm */*.ppm */*.ktx */*.ktx2 */*.basis */*.pspimage
+Copyright: 2015-2020 The Khronos Group Inc.
+License: Apache-2.0
```

Comment:
ccwn2c08.png, g03n2c08.png, color_grid.png and kodim17.png come from outside. The first 2 are from the PNG test suite. The license is at http://www.schaik.com/pngsuite/PngSuite.LICENSE. kodim17.png comes from the Kodak Images repo http://r0k.us/graphics/kodak/.

 I'm struggling to find where color_grid is from. I got it from someone in the 3dformats group. I ping the group to see if I can find out.

camera_camera_BaseColor_16bit.png and FlightHelmet_baseColor.png come from the glTF samples repo](https://github.com/KhronosGroup/glTF-Sample-Models) and are in the public domain.

CesiumLogoFlat_palette.png also comes from the glTF samples repo. The license points to https://github.com/AnalyticalGraphicsInc/cesium/wiki/CesiumTrademark.pdf.

All the  other images & icons fall under the Khronos license.

Reply:
I've dealt with all these except color_grid, which I assigned Apache 2.0 to for the moment.

Reply:
Maybe spin off an issue to track finding the original license on color_grid.png, but I don't think it's essential - if we get the license wrong the the creator can just ask us to change it.